smartcontractkit/smart-contract-db
# feat: adds blockchains page

# Overview

Adds blockchains page

# Screenshots for Mobile and Desktop views (if applicable)
![image](https://user-images.githubusercontent.com/20880360/137040990-cf01a158-2139-4cd9-ad46-aad9d7428dd3.png)

### Manual Testing

Write the steps required for how might test this. Example:

1. Run `npm run dev`
2. In browser, goto `localhost:3000/`
3. Navigate to blockchains and verify data
4. Get results :tada:
